python/TestHarness/TestHarness.py

```diff
@@ -789,6 +804,18 @@ def parseCLArgs(self, argv):
         tmp_str = getattr(self.options, key)
         setattr(self.options, key, value[0])
 
+    # If attempting to test only failed_tests, open the .failed_tests file and create a list object
+    self.test_list = []
+    if self.options.failed_tests:
+      with open(os.path.join(os.getcwd(), '.failed_tests'), 'r') as tmp_failFile:
```

Comment:
`tmp_failFile` is a mixture of C-style and camel case. It should be C-style -> `tmp_fail_file`.